localstack-core/localstack/testing/scenario/cdk_lambda_helper.py

```diff
@@ -81,39 +82,40 @@ def load_nodejs_lambda_to_s3(
     :param key_name: key name for the uploaded zip file
     :param code_path: the path to the source code that should be included
     :param additional_nodjs_packages: a list of strings with nodeJS packages that are required to run the lambda
+    :param additional_nodejs_packages: a list of strings with nodeJS packages that are required to run the lambda
     :param additional_resources: list of path-strings to resources or internal libs that should be packaged into the lambda
     :return: None
     """
     additional_resources = additional_resources or []
 
+    if additional_nodjs_packages:
+        additional_nodejs_packages = additional_nodejs_packages or []
+        additional_nodejs_packages.extend(additional_nodjs_packages)
```

Comment:
nice, great job implementing the discussed pattern to make this backwards compatible for now.

I like that you put this early in its own block, so it's easy to remove again later :+1: